src/main/java/org/jabref/gui/exporter/SaveDatabaseAction.java

```diff
@@ -272,9 +294,9 @@ private void saveWithDifferentEncoding(Path file, boolean selectedOnly, Charset
     private boolean readyForAutosave(BibDatabaseContext context) {
         return ((context.getLocation() == DatabaseLocation.SHARED) ||
                 ((context.getLocation() == DatabaseLocation.LOCAL)
-                        && preferences.getBoolean(JabRefPreferences.LOCAL_AUTO_SAVE)))
-                &&
-                context.getDatabasePath().isPresent();
+                 && preferences.getBoolean(JabRefPreferences.LOCAL_AUTO_SAVE)))
+               &&
```

Comment:
indent is off

Reply:
Nope that's correct, one && belongs to the outer conditions.

Reply:
can you please refactor it by introducing a local variable. This is hard to read and understand.